OrbitClientServices/ProcessManager.cpp

```diff
@@ -202,6 +205,19 @@ ErrorMessageOr<std::string> ProcessManagerImpl::LoadProcessMemory(
   return std::move(*response.mutable_memory());
 }
 
+ErrorMessageOr<std::string> ProcessManagerImpl::LoadNullTerminatedString(
+      int32_t pid, uint64_t address, uint64_t max_size){
+  auto errorOrString = LoadProcessMemory(pid, address, max_size);
```

Comment:
I am not sure this will load null-terminated string. You might want to find first '\0' manually and resize the string.

it looks like you are currently returning just a fragment of memory with size 0, and letting caller figure out what to do with it. In which case this is not better than LoadProcessMemory.